test/client/before-test.js

```diff
@@ -1,4 +1,7 @@
 (function () {
+    $('script').addClass('script-TC2b9a6d');
+    $('link').addClass('ui-stylesheet-TC2b9a6d');
```

Comment:
Let's add the comment which will clarify this behavior then. It's not quite clear currently.